CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ cmake_dependent_option(BUILD_LIBSND "Build with support for libsnd library" ON
 cmake_dependent_option(BUILD_LIBTAR "Build with support for libtar library" ON
                        "NOT BUILD_DALI_NODEPS" OFF)
 option(BUILD_FFTS "Build with ffts support" ON)  # Built from thirdparty sources
+option(BUILD_CFITSIO "Build with cfitsio support" ON)  # Built from thirdparty sources
```

Comment:
If we don't ship the library (either in `third_party` or `DALI_deps`), then we should enable it by default.
Alternatively, we can just try to use it and disable (instead of giving a hard error) when it's not found.

Reply:
I think that the plan is to ship it with DALI_deps (https://github.com/NVIDIA/DALI_deps/pull/68) , although  I don't know if enabling it by default is a way to go since it will be only needed for FITS reader. What do you recon?

Reply:
OK, then add the commit hash (from your GH repo) to this PR - then we can run it in CI and test that it compiles and works. After that we can merge the changes to DALI extra and specify the proper commit hash in this PR. Then it will be mergeable.